ksql-engine/src/main/java/io/confluent/ksql/structured/SqlPredicate.java

```diff
@@ -78,7 +78,8 @@
 
     try {
       ee = CompilerFactoryFactory.getDefaultCompilerFactory().newExpressionEvaluator();
-
+      ee.setDefaultImports((String[]) CodeGenRunner.CODEGEN_IMPORTS.toArray(
```

Comment:
remove cast and use `new String[0]` for more succinct code. Or just add a method on CodeGenRunnder to get the thing as an array.